testsuite/MDAnalysisTests/core/test_groups.py

```diff
@@ -639,62 +639,108 @@ def test_segmentgroup_str(self, u):
         assert_(str(sg) == '<SegmentGroup [<Segment 4AKE>]>')
```

Comment:
If you get to change anything else, there is a lost `assert_` here.

Reply:
There are a lot more of them in the tests. I would like to change those in a separate PR.